src/generic_solver.cpp

```diff
@@ -66,12 +65,19 @@ std::string & trim(std::string & str)
 }
 
 // class methods implementation
-GenericSolver::GenericSolver(string path, vector<string> cmd_line_args, uint write_buf_size, uint read_buf_size)
-  : AbsSmtSolver(SolverEnum::GENERIC_SOLVER), path(path), cmd_line_args(cmd_line_args), write_buf_size(write_buf_size), read_buf_size(read_buf_size),
-  name_sort_map(new unordered_map<string, Sort>()),
-  sort_name_map(new unordered_map<Sort, string>()),
-  name_term_map(new unordered_map<string, Term>()),
-  term_name_map(new unordered_map<Term, string>())
+GenericSolver::GenericSolver(string path,
```

Comment:
Not sure if this is just github but are there a bunch of trailing spaces on this line?

Reply:
I can't see anything :) And this specific diff is due to clang-format.